### PR TITLE
Create Holaquiroga.md

### PR DESCRIPTION
**Pull request by Javier1995**
Quiero aplicar a la legion del mal

### CODE REVIEW

File: aspirantes/Holaquiroga.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## Aplicacion a la legion del mal
```
Comment (pedromayo2020) [inline aspirantes/Holaquiroga.md:1]: ##Hola

what more do you want me to do?

Comment (pedromayo2020) [reply aspirantes/Holaquiroga.md:1]: More conficts

**Review by pedromayo2020 — CHANGES_REQUESTED** [review]

File: aspirantes/Holaquiroga.md (hunk 1 above)
Comment (Javier1995) [reply aspirantes/Holaquiroga.md:1]: Fuck you if you don't want my changes

Comment (pedromayo2020) [reply aspirantes/Holaquiroga.md:1]: Alright!

Comment (pedromayo2020) [reply aspirantes/Holaquiroga.md:1]: add more content

**Review by pedromayo2020 — APPROVED** [review]
Bien lo pasare a revisar

**Pull request merged**